### ISSUE TITLE
End of task bar doesn't reach the end date

### ISSUE THREAD

**Issue by antmanin** [csgoh/roadmapper#69]
The end of task is December 31, 2024
But the bar ends in October
<img width="1376" alt="image" src="https://github.com/csgoh/roadmapper/assets/28486172/cd0ce085-b1fb-457c-bc14-18bfa1187e04">
```roadmap = Roadmap(1400, 1400, colour_theme="BLUEMOUNTAIN")
roadmap.set_title("Title")
roadmap.set_timeline("M", start="2023-01-01", number_of_items=34)

group = roadmap.add_group("Group")

task = group.add_task("Task", "2023-01-01", "2024-12-31", fill_colour='#00FF00')
roadmap.draw()
roadmap.save("demo01.png")```

**Comment by csgoh** [comment]
Thanks for letting me know. It will be fixed.

**Comment by csgoh** [comment]
@antmanin This is now fixed in the latest version 1.3.2.
<img width="848" alt="image" src="https://github.com/csgoh/roadmapper/assets/20867152/6999c8e3-0be0-4627-b10a-d82c953fecbd">

**Comment by antmanin** [comment]
@csgoh it works, thanks!
